components/input/input.scss

```diff
@@ -53,6 +53,14 @@
     border-color: var(--tx-input-border-hover-color);
   }
 
+  &.ui-input_disabled {
+    opacity: 0.5;
```

Comment:
I don't think that applying opacity is a good way of showing disabled state. I rather prefer to play with colors instead.

Reply:
As I have written in PR's note the decision to make such style changes was made by designer Nicolas.

Reply:
I think that we need to discuss that. And if everyone agrees - apply this styling for all disabling states to avoid inconsistency. What do you guys think about semi-transparency for the disabled state? My concern is looking will depend on background, I think that it's not good. We have a lot of specific hues for each theme, better to use one of them for disabled state.

Reply:
@yurist38 when new designs were introduced for the input elements the disabled state was missed out. @annavolkovskaya was talking to the designers and this is for now the solution... therefore until we have other design introduced we will use this approach ;)